app/assets/javascripts/components/privacy.js

```diff
@@ -0,0 +1,312 @@
+              <li>Organization affiliation</li>
+              <li>Encrypted password</li>
+              <li>Activity times</li>
+              <li>Application configurations</li>
+            </ul>
+            This data is attached to the user&apos;s account. It is used to identify the annotation
+            data that was created by the user and for administrative purposes. The legal bases are
+            user consent (upon signup) or legitimate interests in order to provide the Service.
+          </li>
+          <li>
+            Registered users may participate in discussions on the webKnossos discuss platform.
+            Posts and replies that they make there are visible to other webKnossos users (also from
+            other webKnossos instances). Their posts and replies may be removed at any time.
+          </li>
+          <li>
+            Registered users may choose to publish datasets or annotations they have created or
+            uploaded or are maintaining. This data will then be publicly available to everybody on
+            the internet. Personal data such as name, email address and organization affiliation may
+            be made public as well in order to provide the attribution that is common within the
+            scientific community. Users may unpublish their work at any time.
+          </li>
+          <li>
+            For scientific purposes it may be required to retain personal information in order to
+            provide attribution and accountability.
+          </li>
+          <li>
+            We do not sell, trade, share, or rent the personal data collected from our Service to
+            third parties other than as outlined in this policy.
+          </li>
+        </ul>
+
+        <h3>Your rights</h3>
+        <ul>
+          <li>
+            If you wish to confirm, access, update/correct or request deletion of your personal
+            data, you can do so here or by contacting us.
+          </li>
+          <li>
+            You can always object to processing the of your personal data, please ask us to restrict
+            processing of your personal data or request a data export. Again, you can do so here or
+            by contacting us.
+          </li>
+          <li>
+            You have the right to complain to a data protection authority about our collection and
+            use of your personal data. For more information, please contact your local data
+            protection authority.
+          </li>
+          <li>
+            You can request an archive of the data we have stored about you. To file for such a
+            request please login to your account here and submit a data export request.
+          </li>
+        </ul>
+        <h3>Legal basis</h3>
+        <p>
+          Art. 6(1) lit. a GDPR serves as the legal basis for processing operations for which we
+          obtain consent for a specific processing purpose. If the processing of personal data is
```

Comment:
consent ... haben wir noch nicht. sollten wir mal in den signup process aufnehmen